makefiles/test.mk

```diff
@@ -11,8 +11,8 @@ test_unit: ## Run all unit tests
 
 .PHONY: test_e2e_evm_morse
 test_e2e_evm_morse: morse_e2e_config_warning ## Run an E2E Morse relay test
-	(cd ./e2e && TEST_PROTOCOL=morse go test -tags=e2e -count=1 -run Test_PATH_E2E_EVM)
+	DOCKER_LOG=true TEST_PROTOCOL=morse go test -v -tags=e2e -count=1 -run Test_PATH_E2E_EVM ./e2e
```

Comment:
My strong preference would be to **not** include `DOCKER_LOG=true` in the default test make target. It adds a LOT of noise to the log output and IMO adds very little value unless one is actively debugging a Docker build issue.

IMO we should leave this out and allow it to be set as an optional env var, eg you can just run:
`make test_e2e_evm_morse DOCKER_LOG=true` if required as opposed to requiring a developer to choose to set it to false to avoid it.